actions/workflows/archive_upload.yaml

```diff
@@ -0,0 +1,71 @@
+version: "2.0" # mistral version
+name: arteria-packs.archive_upload
+description: Archives a runfolder to PDC with TSM
+
+workflows:
+    main:
+        type: direct
+        input:
+            - runfolder
+            - host
+            - remove_previous_archive
+        output:
+            output_the_whole_workflow_context: <% $ %>
```

Comment:
Perhaps `output` in the beginning of the variable name here is a bit redundant?

Reply:
This is what we call it in all our workflows, so I think I'll let it be like this for now.

Reply:
:+1: